example_plugins/updater_plugin/pytest/test_example_updater.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2009-2022 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+from typing import List, Optional
+import itertools
+import pytest
+
+import hoomd
+from hoomd import operation
+from hoomd.device import Device
+
+from hoomd import updater_plugin
+
+import numpy as np
+
+
+def build_system(vel: Optional[List[float]] = None,
```

Comment:
You could base this on the `one_particle_snapshot_factory` fixture. Here is an example of making a fixture to extend another of the snapshot factories:
https://github.com/glotzerlab/hoomd-blue/blob/73c79a05f517c1c97b95f92702204c4b7f8405f5/hoomd/md/pytest/test_bond.py#L99-L112

Reply:
Yea, I noticed these when I was looking through the existing tests. No problem!